spec/models/course_spec.rb

```diff
@@ -2135,6 +2135,119 @@
     end
   end
 
+  describe "create course notification emails" do
+    let(:provider) { create(:provider) }
+    let(:accrediting_provider) { create(:provider, :accredited_body) }
+    let(:organisation) { create(:organisation, providers: [provider]) }
+    let(:site_status) { create(:site_status, :findable) }
+    let(:user_one) { create(:user, organisations: [organisation]) }
+    let(:user_two) { create(:user, organisations: [organisation]) }
+
+    let(:course) do
+      create(
+        :course,
+        provider: provider,
+        accrediting_provider_code: accrediting_provider.provider_code,
+        age_range_in_years: "11_to_15",
+        qualification: "pgce_with_qts",
+        study_mode: "full_time",
+        site_statuses: [site_status],
+        maths: :equivalence_test,
+        english: :equivalence_test,
+      )
+    end
+
+    let(:mail_spy) { spy }
+    let(:mailer_spy) { spy(course_create_email: mail_spy) }
+
+    before do
+      stub_const("CourseCreateEmailMailer", mailer_spy)
+    end
+
+    context "a self-accredited course" do
+      let(:course) { create(:course, :self_accredited, provider: accrediting_provider) }
+
+      before do
+        UserNotification.new(user: user_one, provider_code: accrediting_provider.provider_code, course_create: true).save!
```

Comment:
We can use the factory here now (and in the rest of this file)

Reply:
Whoops, my find and replace was incomplete. Done now!